.travis.yml

```diff
@@ -11,5 +11,8 @@ deploy:
   on:
     branch: master
 before_install:
-  - openssl aes-256-cbc -K $encrypted_bb8f495d020f_key -iv $encrypted_bb8f495d020f_iv
-    -in app-engine-secret.json.enc -out app-engine-secret.json -d
+  - |
+    if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
```

Comment:
just out of curiosity, where is this env variable set? (i'm assuming it's an env variable)

Reply:
https://docs.travis-ci.com/user/environment-variables/#convenience-variables